src/Classes/Utils/Utils.ts

```diff
@@ -48,6 +59,10 @@ export class Utils {
         return destination;
     }
 
+    public static mixComponents(comp1: number, comp2: number, weight1: number, weight2: number) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'mixComponents' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)